### ISSUE TITLE
Add a flavour without Google Play and Facebook for FLOSS delivery

### ISSUE THREAD

**Issue by TheOneWithTheBraid** [ONLYOFFICE/documents-app-android#15]
For #2, a build flavour without proprietary dependencies is required. This should not be very difficult to implement.

**Comment by fossdd** [comment]
Yes. We can't continue the [RFP](https://gitlab.com/fdroid/rfp/-/issues/1643) with the prop. trackers and libraries. All of the exodus report have to be removed.

**Comment by Andrew-Dudov** [comment]
Hello @TheOneWithTheBraid I suggest you look at the issue https://github.com/ONLYOFFICE/documents-app-android/issues/2
